content/md/what-is/what-about-reference-entities.md

```diff
@@ -0,0 +1,39 @@
+---
+id: what-about-reference-entities
+themes: first-steps
+title: What about **reference entities**?
+popular: true
+ee-only: true
+related:
+---
+
+# What's the use case?
```

Comment:
Regarding the summary of your article, I think it's too linear. 
Here what I would suggest:
# What's the use case?
# A bit of vocabulary
  ## What is a reference entity?
  ## What is a record?
# Let's create your first reference entity

What do you think?

Reply:
Yes 👍